.github/workflows/ci.yml

```diff
@@ -362,6 +362,44 @@ jobs:
             KERNELDIR=/lib/modules/$(ls /lib/modules)/build make scap-open driver bpf unit-test-libsinsp -j6
             ./libsinsp/test/unit-test-libsinsp
 
+  build-libs-ppc64le:
```

Comment:
This is a nice idea! Can we use a `matrix` to merge together this job with the build-libs-s390x one?

Reply:
Yes we can. I will try it out.

Reply:
this is a great point, moreover, I just thought that the s390x job takes almost 1 h to run but in the end, we want to check the build of our drivers (modern_bpf, kmod, bpf) why do we need to build `sinsp` and run tests? In order to not slow down our CI too much i would propose to just build the drivers in this CI job, and maybe we can do the same for the s390x WDYT?

Reply:
Do you recommend a separate job for testing `sinsp` or do we stick to the driver builds for now?

Reply:
I think sticking with the drvers build for now is fine; higher level code rarely needs adjustments for new architecture, therefore we are pretty confident that we cannot break the build of scap/sinsp for a given architecture.

Reply:
yep let's say that breaking the build of the userspace part should never happen and a 1h job at every build seems a high price to pay

Reply:
Makes sense. I will modify for both archs. @hbrueckner let me know if this is ok for `s390x`

Reply:
HI @mdafsanhossain 
> Makes sense. I will modify for both archs. @hbrueckner let me know if this is ok for `s390x`

@Andreagit97 @FedeDP let's reduce the build time on `s390x` by focusing on the drivers only.

Reply:
Also please make sure to share `drivers_test` results on kmod,ebpf and modern ebpf once this is ready, thank you!